lib/livebook/teams/client.ex

```diff
@@ -0,0 +1,66 @@
+defmodule Livebook.Teams.Client do
+  @moduledoc false
+
+  alias Livebook.Teams.Org
+  alias Livebook.Utils.HTTP
+
+  @doc """
+  Send a request to Livebook Team API to create a new org.
+  """
+  @spec create_org(Org.t()) :: {:ok, map()} | {:error, map()} | {:transport_error, String.t()}
```

Comment:
The error can also be `{:error, String.t()}` as in `{:error, body}`?

Reply:
Yeah, nice call! I'll update the specs